db/db.go

```diff
@@ -43,7 +45,8 @@ type DB struct {
 // New returns a new database client that implements the AuthDB interface.
 func New(c *Config) (AuthDB, error) {
 	if c == nil {
-		return new(NoopDB), nil
+		_db := new(SimpleDB)
+		return _db.Init(c)
```

Comment:
Instead of doing this it will be cleaner to do just `newSimpleDB()` or `NewSimpleDB()`